process_report/processors/prepayment_processor.py

```diff
@@ -22,11 +22,24 @@ class PrepaymentProcessor(discount_processor.DiscountProcessor):
     def PREPAY_DEBITS_S3_BACKUP_FILEPATH(self):
         return f"Prepay/Archive/prepay_debits {util.get_iso8601_time()}.csv"
 
+    @property
+    def CREDITS_SNAPSHOT_FILEPATH(self):
+        return f"NERC_Prepaid_Group-Credits-{self.invoice_month}.csv"
+
+    @property
+    def CREDITS_SNAPSHOT_S3_FILEPATH(self):
+        return f"Invoices/{self.invoice_month}/NERC_Prepaid_Group-Credits-{self.invoice_month}.csv"
+
+    @property
+    def CREDITS_SNAPSHOT_S3_ARCHIVE_FILEPATH(self):
+        return f"Invoices/{self.invoice_month}/Archive/NERC_Prepaid_Group-Credits-{self.invoice_month} {util.get_iso8601_time()}.csv"
+
     prepay_credits: pandas.DataFrame
     prepay_projects: pandas.DataFrame
     prepay_contacts: pandas.DataFrame
     prepay_debits_filepath: str
     upload_to_s3: bool
+    export_NERC_credits: bool = True  # For testing purposes
```

Comment:
`export_NERC_credits` is inconsistently capitalized, should be `export_nerc_credits`.